template/readme/local-development.md

```diff
@@ -27,78 +27,8 @@ If you must use Windows, we recommend that:
 
 To use [Drupal VM](https://www.acquia.com/products-services/dev-desktop) with a Drupal project that is generated with BLT:
 
-1. Place your downloaded copy of Drupal VM inside the generated Drupal project folder, and name the drupal-vm directory `box`.
-1. Inside the new project's `build/custom/phing/build.yml` file, override the `docroot` used for Drush commands with the value:
-
-    ```
-    drush:
-      root: ''
-    ```
-1. Add a drush alias to `drush/site-aliases/aliases.drushrc.php`:
-
-    ```
-    // [vagrant_machine_name].local
-    $aliases['[vagrant_machine_name].local'] = array(
-      // /var/www/[vagrant_machine_name]/docroot
-      'root' => '/var/www/[vagrant_machine_name]/docroot',
-      // vagrant_hostname
-      'uri' => '[vagrant_hostname]',
-      // vagrant_hostname
-      'remote-host' => '[vagrant_hostname]',
-      'remote-user' => 'vagrant',
-      'ssh-options' => '-i ~/.vagrant.d/insecure_private_key',
-    );
-    ```
-1. Updating `project.yml` so that `drush.default_alias` is set to your Drupal VM alias.
+1. Execute `./blt.sh vm:init` from the project root directory.
 1. Follow the Quick Start Guide in [Drupal VM's README](https://github.com/geerlingguy/drupal-vm#quick-start-guide)
```

Comment:
Technically it can always be `1` (or any number) since Markdown parses it as a typical `ol` item in the resulting HTML... I like numbering them in the markdown itself because I usually read Markdown in its native format... but many people like using `1 1 1 1 1 1` since it requires renumbering if you're off (well, technically you could have any number there, but I'm a little OCD, so I always renumber when editing) :D